mpcpy/exodata.py

```diff
@@ -1332,7 +1337,101 @@ def _collect_data(self, start_time, final_time):
+             time resolution: 1 hour, geographical resolutions: 3 km
+        RAP: Rapid Refresh model, available US and for 2 days ahead (tested 18 hours), support historical data, updated every hour, 
+             time resolution: 1 hour, geographical resolutions: 20, 40 km
+        NAM: North American Mesoscale model, available for the whole North America and for 4 days ahead (tested 3 days), support historical data, 
+             updated every 6 hours, time resolution: 1 hour, geographical resolutions: 20 km
+    tz_name: timezone name, str
+        Default is 'from geography', then the timezone would be inferred automatically from the input geography
+        Users could also specify the timezone manually, example 'America/Chicago', 'UTC'
+
+    Attributes
+    ----------
+    data : dictionary
+        {"Weather Variable Name" : mpcpy.Variables.Timeseries}.
+    lat : mpcpy.variables.Static
+        Latitude in degrees.
+    lon : mpcpy.variables.Static
+        Longitude in degrees.
+    tz_name : string
+        Timezone name.  
+
+    '''
+    
+    def __init__(self, geography, weaForeModel, tz_name='from_geography', **kwargs):
+        '''Constructor of DataFrame weather exodata object.
+        
+        '''
+        
+        self.name = 'weather_from_noaa';
+        self.data = {};   
+        # Dictionary of format {'dfHeader' : ('weaVarName', mpcpyUnit)}
+        self.variable_map = {'temp_air'     : ('weaTDryBul', units.degC),
+                             'wind_speed'   : ('weaWinSpe', units.m_s),
+                             'ghi'          : ('weaHGloHor', units.W_m2),
+                             'dni'          : ('weaHDirNor', units.W_m2),
+                             'dhi'          : ('weaHDifHor', units.W_m2),
+                             'total_clouds' : ('weaNTot', units.percent),
+                             };
+        self.geography = geography
+        self.lat = variables.Static('lat', self.geography[0], units.deg);
+        self.lon = variables.Static('lon', self.geography[1], units.deg)
+        
+        # Initialize the weather forecast model
+        if weaForeModel == 'GFS':
+            self.model = GFS()
+        elif weaForeModel == 'HRRR':
+            self.model = HRRR()
+        elif weaForeModel == 'RAP':
+            self.model = RAP()
+        elif weaForeModel == 'NAM':
+            self.model = NAM()
+        else:
+            raise NameError('The {} forecast model is not supported. Only GFS, HRRR, RAP, NAM are supported now'.format(method))
+        
+        self._parse_daq_kwargs(kwargs)
+        if tz_name == 'from_geography':
+            self.tz = tzwhere.tzwhere();
+            self.tz_name = self.tz.tzNameAt(self.geography[0], self.geography[1]);
+        else:            
+            self.tz_name = tz_name;
+           
+    def _collect_data(self, start_time_local, final_time_local):
```

Comment:
To be consistent with other MPCPy classes for exodata collection, ``start_time`` and ``final_time`` should be assumed to be in whatever timezone is defined in the geography constructor parameter.  Please remove ``_local``.